<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createAccount</class>
 <widget class="QDialog" name="createAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Account</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>:/paborito Logo.png</activeon>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Confirm">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Kozuka Gothic Pr6N B</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 0);</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Kozuka Gothic Pr6N B</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Full Name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_FirstName">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="text">
             <string>First Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_MiddelName">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="text">
             <string>Middle Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_LastName">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="text">
             <string>Last Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_FirstName">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_MiddelName">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lineEdit_LastName">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <family>Kozuka Gothic Pr6N B</family>
      </font>
     </property>
     <property name="title">
      <string>User Account</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_Username">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_Password">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_Type">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="text">
             <string>Account Type</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEdit_Username">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit_Password">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="comboBox_accountType">
            <property name="font">
             <font>
              <family>Kozuka Gothic Pr6N B</family>
             </font>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Manager</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cashier</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Kozuka Gothic Pr6N B</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 127);</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
